apps/storybook/src/WorkflowDiagram.test.ts

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+describe('WorkflowDiagram', () => {
```

Comment:
Is WorkflowDiagram a completely separate component now?

Reply:
`WorkflowDiagram` actually already existed. But `Wizard` was a wrapper around `WorkflowDiagram`.

So people earlier had to do `<Wizard type:'workflow''/>` to get `WorkflowDiagram`. Now they can directly use `WorkflowDiagram`

You can see [L59-70](https://github.com/iTwin/iTwinUI-react/pull/905/files#diff-31a31bcd2c19607b8a9dfdfdd6da3ac2990b7dfd28d59ec85314055793880fa8R59-R70) to get a better idea.

Reply:
I guess my question is... do you know why we moved WorkflowDiagram out from under Wizard (now Stepper)? 
Why don't we do `<Stepper type="workflow"/>`?

Reply:
I thought since we are deprecating `Wizard`, I could expose `Stepper` and `WorkflowDiagram`. But maybe I could somehow combine `Wizard` into `Stepper` to make it just like a rename change `Wizard` -> `Stepper` on the users end (since that might be helpful for users to easily update to v2)

Will confirm which approach to take.

Reply:
I saw the discussion, makes sense to me. Just wanted to check.